app/assets/stylesheets/modules/parts_section.scss

```diff
@@ -1,5 +1,15 @@
 .parts-section {
   padding-left: 0;
+
+  [data-toggle="collapse"] {
```

Comment:
Do you think we should try to decide on a common pattern for this and the metadata details section?

Reply:
Most definitely.. Updated with consolidation of this.